tests/fixtures/Payments/can_get_payment_confirmation.yaml

```diff
@@ -0,0 +1,13 @@
+-
+    request:
+        method: GET
+        url: 'https://devapi.currencycloud.com/v2/payments/796e0d7d-bae6-4d8a-b217-3cf9ee80a350/confirmation'
+        headers:
+            X-Auth-Token: e5070d4a16c5ffe4ed9fb268a2a716be
+    response:
+        status: 200
+        headers:
+            Date: Wed, 31 Oct 2018 09:10:59 GMT
+            Content-Type: application/json
+            X-Request-Id: 'ccb90187-72de-470d-bda5-d08e8833459b'
+        body: '{"id":"e6b30f2d-0088-4d99-bb47-c6b136fcf447","payment_id":"796e0d7d-bae6-4d8a-b217-3cf9ee80a350","account_id":"72970a7c-7921-431c-b95f-3438724ba16f","short_reference":"PC-2436231-LYODVS","status":"failed","confirmation_url":"","created_at":"2018-08-03T13:18:23+00:00","updated_at":"2018-08-03T13:18:23+00:00","expires_at":null}'
```

Comment:
A successful payment confirmation must return a not null confirmation_url and a success status. Test must reflect this conditions

Reply:
@EAddario I've updated the tests.